src/request/component-models/LoggingComponentModel.ts

```diff
@@ -1,39 +1,164 @@
 'use strict';
 
-/*tslint:disable:no-var-requires */
-const messageProcessor = require('../util/request-message-processor');
-/*tslint:enable:no-var-requires */
-
-import { ILoggingComponentModel } from './ILoggingComponentModel';
+import { ComponentModel } from './ComponentModel';
+import { ILoggingComponentModel, ILoggingComponentState, ILoggingLevelModel, ILogMessageModel } from './ILoggingComponentModel';
 import { ILogMessage } from '../messages/ILogMessage';
 import { IMessageEnvelope } from '../messages/IMessageEnvelope';
 
 import _ = require('lodash');
 
-export class LoggingComponentModel implements ILoggingComponentModel {
-    private static getList = messageProcessor.getTypeMessageList;
+class LogMessageModel implements ILogMessageModel {
+    public constructor(private _message: IMessageEnvelope<ILogMessage>, private _ordinal: number) {
+    }
+
+    public get id(): string {
+        return this._message.id;
+    }
+
+    public get level(): string {
+        return this._message.payload.level;
+    }
+
+    public get message(): string {
+        return this._message.payload.message;
+    }
 
-    private static options = {
-        'log-write': LoggingComponentModel.getList
-    };
```

Comment:
Any particular reason this got pulled from being a static to inside the init method?